src/python/turicreate/toolkits/image_similarity/image_similarity.py

```diff
@@ -41,8 +41,8 @@ def create(dataset, label = None, feature = None, model = 'resnet-50', verbose =
         identify reference dataset rows when the model is queried.
 
     feature : string
-        indicates that the SFrame has only column of Image type and that will
         Name of the column containing the input images. 'None' (the default)
+        indicates that the SFrame has only column of Image type and that will
```

Comment:
Should this be "SFrame has only **one** column of Image type"?

Reply:
Probably the word *one* is needed here, since it calls `_find_only_image_column`, which finds a column of type `Image` and raises and exception if there is more than one. Changing.